src/app/development/middleware.ts

```diff
@@ -112,6 +112,14 @@ export const templatesDisplayGet: RequestHandler = async (req, res) => {
   } else if (templateId === PAGES.PYI_TRIAGE_MOBILE_DOWNLOAD_APP) {
     validatePhoneType(phoneType);
     renderOptions.appDownloadUrl = getAppStoreRedirectUrl(phoneType);
+  } else if (templateId === PAGES.PAGE_FACE_TO_FACE_HANDOFF) {
+    let today = new Date();
+    if (process.env.NODE_ENV === "local") {
+      today = new Date("2025-01-01");
+    }
+    renderOptions.postOfficeVisitByDate = new Date().setDate(
+      today.getDate() + 15,
+    );
```

Comment:
could we have this just always hardcode the date (rather than relying on NODE_ENV being set to local) ?
should be fine for just a preview of the page